examples/material_gallery/test_driver/scroll_perf_test.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('scrolling performance test', () {
+    FlutterDriver driver;
+
+    setUpAll(() async {
+      driver = await FlutterDriver.connect();
+    });
+
+    tearDownAll(() async {
+      if (driver != null)
+        driver.close();
+    });
+
+    test('measure', () async {
+      Timeline timeline = await driver.traceAction(() async {
+        // Find the scrollable stock list
+        ObjectRef stockList = await driver.findByValueKey('Gallery List');
+        expect(stockList, isNotNull);
+
+        await driver.tap(await driver.findByText('Demos'));
+        await driver.tap(await driver.findByText('Components'));
+        await driver.tap(await driver.findByText('Style'));
+
+        // Scroll down
+        for (int i = 0; i < 5; i++) {
+          await driver.scroll(stockList, 0.0, -300.0, new Duration(milliseconds: 300));
+          await new Future<Null>.delayed(new Duration(milliseconds: 500));
```

Comment:
And that's copypasta from my code :) If you think of scrolling as a sequence of "pointer down", "drag", "pointer up", "move finger back to the original location", "pointer down", "drag", etc, then this pause emulates the "move finger back to the original location" part. The duration of 500ms is, just like the scroll duration above it, is not based on any empirical data. It was just me testing the testing framework. Also, `scroll` does not emulate the acceleration/deceleration of a user's finger. We need to do some homework in this area.